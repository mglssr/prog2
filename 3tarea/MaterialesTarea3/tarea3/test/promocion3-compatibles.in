crearPromocion 01/01/2002 01/06/2002 10 1
agregarAPromocion 5 0 1 2 3 4
imprimirPromocion
sonPromocionesCompatibles 01/01/2002 01/06/2002 10 2 5 0 1 2 3 4
sonPromocionesCompatibles 01/01/2002 01/06/2002 10 2 5 5 6 7 8 9
liberarPromocion
Fin
