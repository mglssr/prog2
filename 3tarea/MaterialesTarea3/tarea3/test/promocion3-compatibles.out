Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> Promocion creada con exito.
2> Productos agregados a la promocion de forma exitosa.
3> Promocion #1 del 1/1/2002 al 1/6/2002
Productos: 0 1 2 3 4 
4> Las promociones NO son compatibles.
5> Las promociones son compatibles.
6> Promocion liberada con exito.
7> Fin.
